.travis.yml

```
@@ -31,6 +31,7 @@ env:
         - CONDA_CHANNELS='astropy-ci-extras astropy glueviz'
         - PIP_DEPENDENCIES='matplotlib<2 Cython https://github.com/radio-astro-tools/pvextractor/archive/master.zip radio_beam https://github.com/astropy/regions/archive/master.zip'
```

If it still segfaults, then try this.
```suggestion
        - PIP_DEPENDENCIES='pytest-faulthandler matplotlib<2 Cython https://github.com/radio-astro-tools/pvextractor/archive/master.zip radio_beam https://github.com/astropy/regions/archive/master.zip'
```